spot_rl_experiments/spot_rl/utils/img_publishers.py

```diff
@@ -686,8 +686,8 @@ def inference(self, hand_rgb, timestamp, stopwatch):
         # Add new classes here to the model
         # We decide to hardcode these classes first as this will be more robust
         # and gives us the way to control the detection
-        # multi_classes = [["ball", "cup", "table", "cabinet", "chair", "sofa"]]
-        multi_classes = rospy.get_param("/multi_class_object_target").split(",")
+        multi_classes = ["ball", "cup"]
+        # multi_classes = rospy.get_param("/multi_class_object_target").split(",")
```

Comment:
I would like to be used with `rospy.get_param("/multi_class_object_target")` to set the object name